roboteam_world/src/main.cpp

```diff
@@ -1,10 +1,60 @@
 #include "Handler.h"
 
+std::optional<std::string> findFlagValue(int argc, char** argv, std::string flag){
+    // Search for flag
+    auto it = std::find(argv, argv + argc, flag);
```

Comment:
Please don't use the old c style with argv and argc. I'd much rather have you read it into a std::vector<std::string> once ( use std::vector<std::string> vector(argv,argv+argc) instead), and then pass that to this function.

Additionally, the flag is constant so you can const ref it.